### ISSUE TITLE
"Nonexistent signal" error message on every single frame in Godot4.3

### ISSUE THREAD

**Issue by kobligo** [EIRTeam/EIRTeam.FFmpeg#35]
I'm using the latest official stable release of Godot4.3 along with EIRTeam.FFmpeg Auto-Build 2024-06-07 on a Windows11 system.
This extension produces the following error message on every single frame that I'm playing a video stream. (Tested with mp4 and webm.)
```
E 0:00:01:0366   connect: In Object of type 'RefCounted': Attempt to connect nonexistent signal 'return_frame' to callable '<CallableCustom>'.
  <C++ Error>    Condition "!signal_is_valid" is true. Returning: ERR_INVALID_PARAMETER
  <C++ Source>   core/object/object.cpp:1390 @ connect()
```

On top of that it prints the number zero to the console on every single frame. Might be related.

**Comment by shoyguer** [comment]
Same problem. I tested only with .mp4 files.
For me, instead of the number zero, is prints the number "5".

Third image is for where the problem is the source code Godot is pointing at.

Godot 4.3, Windows 11
Addon version:
Auto-Build 2024-06-07 02:57

![image](https://github.com/user-attachments/assets/4bfb5074-3457-4ee4-9d1a-956fc80c72dd)

![image](https://github.com/user-attachments/assets/999eb348-56ed-4af1-947b-e58a8c0d89c3)

![image](https://github.com/user-attachments/assets/0f9acff5-db54-4966-96e8-b67bced952d4)

Godot Source line:
https://github.com/godotengine/godot/blob/77dcf97d82cbfe4e4615475fa52ca03da645dbd8/core/object/object.cpp#L1390

**Comment by shoyguer** [comment]
How do I fix that?
This plugin is perfect for my project, I'm currently not using it because of this error 
Thanks for your time

**Comment by Avatarchik** [comment]
Hi!
Rendering method
must be  forward+ or Mobile.
Doesn't work Compatibility

**Comment by koenheltzel** [comment]
I'm just user too, but maybe you can try again with the latest release, I had similar issues that were fixed in #40

**Comment by EIREXE** [comment]
Oh yeah I forgot to mention it, it's fixed now.
